index.html

```diff
@@ -693,9 +693,20 @@
               </li>
               <li>Determine which <a>payment handlers</a> support any of the
               <a>payment method identifiers</a> given <var>identifiers</var>.
-              For each resulting payment handler, if payment method specific
-              capabilities supplied by the payment handler match those provided
-              by <var>data</var>, the payment handler matches.
+              </li>
+              <li>For each <var>handler</var> of the resulting payment
+              handlers:
+                <ol>
+                  <li>If required by the specification that defines the
+                  <var>handler</var>, <a data-cite=
+                  "!WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
+                  <var>data</var> to an IDL value. Rethrow any exceptions.
```

Comment:
Sure, if the data is invalid for *all* payment apps, then the next step (which already exists) will fail. But we need to give every payment app a chance. If one succeeds, but 3 fail, it's not invalid data.